lib/active_merchant/billing/gateways/pay_trace.rb

```diff
@@ -200,6 +200,16 @@ def customer_id?(payment_or_customer_id)
         payment_or_customer_id.class == String
       end
 
+      def to_boolean(value)
```

Comment:
```
to_boolean(true)
=> nil

to_boolean(nil)
=>nil
```

I don't know if these are scenarios that we need to worry about or not. We probably don't have to worry about `nil` because you are only calling `to_boolean` if the field is present. But could the field ever come in as a boolean already? In that case `to_boolean` will return `nil` instead of the value we expect.

These cases might not matter for this gateway. But I do think the name of the method is misleading. Maybe something like `def string_literal_to_boolean`? Or `def convert_true_or_false_string`? Not very charismatic but at least it would force anyone who might use the method to look closely at what it does. I just worry about calling it `to_boolean` in case somebody else ends up using it in a way that's not intended.

If it turns out you do want to handle more cases than the two string literals, you could do something like this:

```
def to_boolean(value)
  value.to_s.downcase == 'true'
end

to_boolean(true)
=> true

to_boolean(nil)
=> false
```

...but then any random value in there, like 'test', will return `false` and maybe that's not what you want.